app/controllers/journey_controller.rb

```diff
@@ -47,4 +42,8 @@ def game
     @location_id = params[:location_id]
     render "journey/game#{@location_id}.html.erb"
   end
+
+  def help
+    render layout: 'wide',  :locals => {:background => "granite"}
```

Comment:
Unnecessary spacing detected.<br>Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.